environs/imagemetadata/validation.go

```diff
@@ -33,9 +30,11 @@ func ValidateImageMetadata(params *simplestreams.MetadataLookupParams) ([]string
 			Endpoint: params.Endpoint,
 		},
 		Series: []string{params.Series},
-		Arches: params.Architectures,
```

Comment:
Yeah, we do. I had it in the conditional couple of more lines down. But Andrew has rightly pointed out that conditional is not needed. 
Anyway, the logic is there.